dynamiqs/rand.py

```diff
@@ -0,0 +1,34 @@
+from jax import numpy as jnp, Array
+import jax
+
+from dynamiqs.utils import dag  # todo: clean this dependency
+
+
+def complex_matrix(dims: tuple[int, ...], key=None) -> Array:
+    key = key if key is not None else jax.random.PRNGKey(0)
+    key1, key2 = jax.random.split(key, 2)
+    mat = jax.random.normal(key1, dims) + 1j * jax.random.normal(key2, dims)
+    return mat
+
+
+def hermitian(n: int, key=None) -> Array:
```

Comment:
```suggestion
def hermitian_matrix(dims: tuple[int, ...], key=None) -> Array:
```
For consistency with `complex_matrix`.

Reply:
Same comment for `dm` and `ket`.

Reply:
It's just that `complex` is a reserved keywork of Python

Reply:
My comment was also about the typing of the `dims` argument. You use `dims` in `complex_matrix`, and `n` in other functions.